search/src/KBMOSearch.cpp

```diff
@@ -573,6 +573,18 @@ pixelPos KBMOSearch::getTrajPos(const trajectory& t, int i) const {
     }
 }
 
+std::vector<pixelPos> KBMOSearch::getTrajPositions(trajectory& t) const
```

Comment:
nit: I'm unfamiliar with the best practices for naming in C++, but these function names seem difficult to tell apart. 
```
getTrajPositions
getTrajPos
getTrajectoryPos
getTrajectoryPosBC
```

Reply:
Good call. I renamed the functions to clarify a bit.